app/helpers/users_helper.rb

```diff
@@ -1,8 +1,9 @@
 module UsersHelper
+	
 	def gravatar_for(user, options = { size: 80 })
-	gravatar_id = Digest::MD5::hexdigest(user.email.downcase)
-	size = options[:size]
-	gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}?s=#{size}"
-	image_tag(gravatar_url, alt: user.name, class: "gravatar")
-end
+		gravatar_id = Digest::MD5::hexdigest(user.email.downcase)
+		size = options[:size]
+		gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}?s=#{size}"
```

Comment:
indentation

Reply:
@PopaFlorian - this still seems to be broken